integrations/tensorflow/e2e/keras/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+# Copyright 2020 Google LLC
```

Comment:
nit: new files can start using 2021 as the copyright year.

The copyright year should be the year the file was _created_ (which could be a while before it is sent for review or merged). Years should _not_ be updated when a file is moved, edited, or copied.

Reply:
Copy/paste has failed me.